### DISCUSSION TITLE
Add to combobox?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [tui-cs/Terminal.Gui#2716] [General]
Hi.
I have been looking around in the documentation and other places, but i have not found a way to add a bunch of stuff to the combobox ( In this case COm port Names).
combobox.Items is not availble in this context.

**Comment by Nutzzz** [comment]
You want .SetSource

As a shortcut, an IList can be used in the constructor.

**Reply by tznind** [reply]
Yup, for example:  

```csharp
comboBox.SetSource(new List<string>() { "Cat", "Dog", "Fish" });
```

One thing to note that is quite important is that he size of a combo box dictates how big the dropdown is.  So if you make it 10x1 then you wont see the dropdown (i.e. because it has only one height).

**Reply by GronbergInfo** [reply]
Thanks, ill look into this.

**Reply by GronbergInfo** [reply]
Interesting about the height. Have to look into how different heights plays into my program.
